regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
             "type": "boolean",
         },
     },
+    "projecta": {
+        "_description": {
+            "description": "This collection describes a single deliverable "
+                            "of a larger project."
+        },
+        "_id": {
+            "description": "Unique projecta identifier",
+            "required": True,
+            "type": "string"
+        },
+        "begin_date": {
+            "description": "projecta start date, yyyy-mm-dd",
+            "required": False,
+            "type": "string"
+        },
+        "collaborators": {
+            "description": "list of collaborator's id's",
```

Comment:
"list of collaborators ids.  These are non-group members. These will be dereferenced from the contacts collection.  Group members will be dereferenced from the people collection."